packages/editor/src/client/blocks/types.ts

```diff
@@ -392,6 +417,91 @@ export function unionMapForBlock(block: BlockValue): BlockInput {
       break
     }
 
+    case BlockType.TeaserGridFlex:
+      return {
+        teaserGridFlex: {
+          flexTeasers: block.value.flexTeasers.map(flexTeaser => {
+            switch (flexTeaser.teaser?.type) {
+              case TeaserType.Article:
+                return {
+                  teaser: {
+                    article: {
+                      style: flexTeaser.teaser.style,
+                      imageID: flexTeaser.teaser.image?.id,
+                      preTitle: flexTeaser.teaser.preTitle || undefined,
+                      title: flexTeaser.teaser.title || undefined,
+                      lead: flexTeaser.teaser.lead || undefined,
+                      articleID: flexTeaser.teaser.article.id
+                    }
+                  },
+                  alignment: {
+                    i: flexTeaser?.alignment.i ?? nanoid(),
+                    x: flexTeaser?.alignment.x ?? 1,
+                    y: flexTeaser?.alignment.y ?? 1,
+                    w: flexTeaser?.alignment.w ?? 1,
+                    h: flexTeaser?.alignment.h ?? 1,
+                    static: flexTeaser?.alignment.static ?? false
```

Comment:
i think you don't need the `?` and the `??` here. TypeScript should know about those values.